fendermint/app/options/src/genesis.rs

```diff
@@ -149,3 +149,34 @@ pub struct GenesisIpcGatewayArgs {
     #[arg(long, short = 'v', default_value = "100")]
     pub active_validators_limit: u16,
 }
+
+#[derive(Args, Debug, Clone)]
+pub struct GenesisFromParentArgs {
+    /// Child subnet for with the genesis file is being created
+    #[arg(long, short)]
+    pub subnet_id: SubnetID,
+
+    /// Endpoint to the RPC of the child subnet's parent
+    #[arg(long, short)]
+    pub parent_endpoint: url::Url,
+
+    /// IPC gateway of the parent
```

Comment:
```suggestion
    /// IPC gateway of the parent; 20 byte Ethereum address in 0x prefixed hex format
```

Reply:
Maybe worth mentioning how one can figure this out when they have to deal with Lotus.